core/pom.xml

```diff
@@ -90,6 +90,13 @@
 			<version>1.0.3</version>
 			<scope>compile</scope>
 		</dependency>
+		<dependency>
+			<groupId>ml.dmlc</groupId>
+			<artifactId>xgboost4j</artifactId>
+			<version>0.5</version>
+			<scope>system</scope>
+			<systemPath>${basedir}/lib/xgboost4j-0.5-jar-with-dependencies.jar</systemPath>
+		</dependency>
```

Comment:
Currently, no `xgboost4j` exists in the maven repository. I'll ask the author later why the library is not uploaded there.